.circleci/mark_stale_endpoint_for_deletion.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+
+# this is complicated because gnu date and BSD date have different implementations
+STALEDATE=$(date --version &>/dev/null && date --date="5 days ago" +"%Y-%m-%d" || date -v-5d +"%Y"-"%m"-"%d")
```

Comment:
same here as before.

Reply:
this was here, but not used. removed it.